### PR TITLE
Fix adding ipset entry comment

### PR DESCRIPTION
**Pull request by DrGonzo**

### CODE REVIEW

**Comment by DrGonzo** [comment]
@vishvananda @aboch could you help me with this pr?

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,6 @@ module github.com/vishvananda/netlink
 go 1.12
 
 require (
-	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae
-	golang.org/x/sys v0.0.0-20220804214406-8e32c043e418
+	github.com/vishvananda/netns v0.0.1
+	golang.org/x/sys v0.2.0
```
Comment (aboch) [inline go.mod:7]: This set of gomod changes look unrelated.
Can you push them through a separate PR?

Comment (DrGonzo) [reply go.mod]: Sorry.
Done

Comment (DrGonzo) [reply go.mod]: @aboch 👋

**Comment by aboch** [comment]
pls squash your commits into one
this project follow a 1 commit per PR rule

**Comment by DrGonzo** [comment]
> pls squash your commits into one
done

**Comment by DrGonzo** [comment]
@aboch done
